src/components/_modal/pages/bookmarkOrder/multi-order/checkoutForm/MultiOrderCheckoutForm.js

```diff
@@ -0,0 +1,223 @@
+        >
+          <Text tag="span" type="text3" className={localizationStyles.fullLink}>
+            {Translate({
+              context: "order",
+              label: "change-pickup-link",
+            })}
+          </Text>
+          <Text
+            tag="span"
+            type="text3"
+            className={localizationStyles.shortLink}
+          >
+            {Translate({
+              context: "general",
+              label: pickupBranch ? "change" : "select",
+            })}
+          </Text>
+        </IconLink>
+      </div>
+      {(isLoadingBranches || pickupBranch) && (
+        <div>
+          <Text type="text3" skeleton={!pickupBranch?.postalAddress} lines={2}>
+            {pickupBranch?.postalAddress}
+          </Text>
+          <Text
+            type="text3"
+            skeleton={!pickupBranch?.postalCode || !pickupBranch?.city}
+            lines={0}
+          >{`${pickupBranch?.postalCode} ${pickupBranch?.city}`}</Text>
+        </div>
+      )}
+    </div>
+  );
+};
+
+const CheckoutForm = ({ context, materialCounts, onSubmit }) => {
+  const { digitalMaterials, materialsNotAllowed, materialsMissingAction } =
+    materialCounts;
+  const disabled = materialsMissingAction > 0 || materialsNotAllowed > 0;
+  const [mail, setMail] = useState(null);
+  const { userInfo, pickupBranchInfo, accessTypeInfo } =
+    useOrderPageInformation({
+      workId: "",
+      periodicaForm: context?.periodicaForm,
+      pids: [],
+    });
+  const { pickupBranch, pickupBranchUser, isLoadingBranches } =
+    pickupBranchInfo;
+
+  const { updateLoanerInfo } = userInfo;
+
+  const validated = useMemo(() => {
+    const hasMail = !!mail?.valid?.status;
+    const hasBranchId = !!pickupBranch?.branchId;
+    const status = hasMail && hasBranchId;
+
+    const details = {
+      hasMail: {
+        status: hasMail,
+        value: mail?.value,
+        message: mail?.valid?.message,
+      },
+      hasBranchId: { status: hasBranchId },
+    };
+
+    return { status, hasTry: false, details };
+  }, [mail, pickupBranch, context?.periodicaForm?.publicationDateOfComponent]);
+
+  const onSubmitForm = () => {
+    console.log(pickupBranch);
+    if (onSubmit) onSubmit(pickupBranch.name);
+  };
+
+  return (
+    <div className={styles.container}>
+      <LocalizationInformation
+        pickupBranch={pickupBranch}
+        pickupBranchUser={pickupBranchUser}
+        accessTypeInfo={accessTypeInfo}
+        isLoadingBranches={isLoadingBranches}
+      />
+      <OrdererInformation
+        context={context}
+        validated={validated}
+        failedSubmission={false}
+        onMailChange={(e, valid) => {
+          onMailChange(e?.target?.value, valid, updateLoanerInfo, setMail);
+        }}
+      />
+
+      <div>
+        {/* Errors and messages */}
+        {materialsNotAllowed > 0 && (
+          <Text type="text3" className={styles.errorLabel}>
+            {materialsNotAllowed ? (
```

Comment:
label= { materialsNotAllowed ? "multiorder-cant-order-singular" : "multiorder-cant-order"}

Reply:
samme i de næste to oversættelser